hw01_hello_now/main.go

```diff
@@ -1,5 +1,20 @@
 package main
 
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/beevik/ntp"
+)
+
 func main() {
-	// Place your code here
+	fmt.Printf("current time: %s\n", time.Now().Round(time.Second))
```

Comment:
https://golang.org/pkg/time/

`The canonical way to strip a monotonic clock reading is to use t = t.Round(0).`